.ci/.fmf/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
what is the purpose of this file ? is this required to add ?

Reply:
Yes, it's is required so that tmt reads this dir. This file exists in the parent dir too, but tmt doesn't read the plan if it is stored in a hidden file. And I want it to be stored as a hidden file so that we do not need to remove it manually when building RPMs (we remove all hidden files automatically.

Reply:
i.e. creating `./.general.fmf` doesn't work. As ell as creating `./.github/general.fmf`

Reply:
Ok I am storing it in the `plans` directory to be consistent with what tmt usually does.
Also, changed the name to be more descriptive.